internal/binder/function/funcs_agg.go

```diff
@@ -24,6 +25,217 @@ import (
 	"github.com/lf-edge/ekuiper/pkg/cast"
 )
 
+func max(a, b float64) float64 {
+	if a > b {
+		return a
+	}
+	return b
+}
+
+func min(a, b float64) float64 {
+	if a < b {
+		return a
+	}
+	return b
+}
+
+func registerGlobalAggFunc() {
+	builtins["acc_avg"] = builtinFunc{
```

Comment:
I think these functions are indeed analytic(stateful) function which keep and update states. Is it better to put them in the analytic functions? Also, they need to support ON Partion By/When clause like the normal analytic functions

Reply:
updated.